test/recipes/controls/aws_parallelcluster_slurm/config_health_check_spec.rb

```diff
@@ -51,9 +51,16 @@
 
   if instance.graphic?
     if instance.dcgmi_gpu_accel_supported?
-      describe command("#{slurm_install_dir}/etc/pcluster/.slurm_plugin/scripts/health_checks/gpu_health_check.sh") do
-        its('exit_status') { should eq 0 }
-        its('stdout') { should match /The GPU Health Check succeeded/ }
+      if os_properties.arm? && (os_properties.centos7? || os_properties.alinux2?)
+        describe command("#{slurm_install_dir}/etc/pcluster/.slurm_plugin/scripts/health_checks/gpu_health_check.sh") do
+          its('exit_status') { should eq 0 }
+          its('stdout') { should match /The Gpu Health Check has been executed but the NVIDIA DCGM Diagnostic tool is not available in this system/ }
+        end
+      else
+        describe command("#{slurm_install_dir}/etc/pcluster/.slurm_plugin/scripts/health_checks/gpu_health_check.sh") do
+          its('exit_status') { should eq 0 }
+          its('stdout') { should match /The GPU Health Check succeeded/ }
```

Comment:
minor: here you have GPU, in the previous message you have Gpu, we should be consistent to simplify parsing of error messages.